hud/CurrencySilverCoin.gd

```diff
@@ -0,0 +1,9 @@
+extends HBoxContainer
+
+@onready var database = get_node("/root/Database")
+
+func _ready():
+	$Amount.text = str(database.silver_coin_count)
+
+func _process(delta):
+	$Amount.text = str(database.silver_coin_count)
```

Comment:
Same thing here - have the DB emit events to handle reflection instead of checking every frame.

Reply:
Fixed via [6696b21](https://github.com/SamBumgardner/packrat-game/pull/1/commits/6696b2192ea3d9b1c4f1da7434c1318879aeea89).